src/Store.js

```diff
@@ -20,6 +20,7 @@ const persistConfig = {
     'user',
     'prescription',
     'patient',
+    'finalise',
```

Comment:
`{ finaliseModalOpen: false, finaliseItem: null }`

I think that reducers should always have an `initialState` function which lists ALL values in the shape - then this can be self-documenting for the shape, what you think?